src/stan_math_backend/Lower_expr.ml

```diff
@@ -241,6 +251,8 @@ and lower_operator_app op es_in =
   | Minus -> lower_binary_op Subtract "stan::math::subtract" es
   | Times -> lower_binary_op Multiply "stan::math::multiply" es
   | Divide | IntDivide ->
+      (* XXX: This conditional is probably a sign that we need to rethink how we store Operators
```

Comment:
What is `XXX`? Should this just be `NOTE: `

Reply:
We've used `XXX` as basically a `FIXME`, e.g. 

https://github.com/stan-dev/stanc3/blob/9e9953f6f168509f1d40a476ce439741508a34fb/src/middle/Internal_fun.ml#L14-L15

https://github.com/stan-dev/stanc3/blob/9e9953f6f168509f1d40a476ce439741508a34fb/src/middle/Stan_math_signatures.ml#L179-L180

I'm not sure when it started, just that I'm pretty sure it was before my time.